examples/resources/awscc_cloudwatch_composite_alarm/with_actions_suppressor.tf

```diff
@@ -0,0 +1,42 @@
+resource "awscc_cloudwatch_composite_alarm" "example" {
+  alarm_name        = "example-composite-alarm"
+  alarm_description = "Example of a composite alarm with actions suppressor"
+
+  actions_suppressor                  = aws_cloudwatch_metric_alarm.cpu_gte_80.alarm_name
+  actions_suppressor_extension_period = 60
+  actions_suppressor_wait_period      = 60
+
+  alarm_actions = [aws_sns_topic.example_alarm_actions.arn]
+
+  alarm_rule = "ALARM(${aws_cloudwatch_metric_alarm.cpu_gte_80.alarm_name}) OR ALARM(${aws_cloudwatch_metric_alarm.status_gte_1.alarm_name})"
+}
+
+resource "aws_sns_topic" "example_alarm_actions" {
+  name = "example-alarm-actions"
+}
```

Comment:
You may want to remove this block of code to keep the core resource simple.

Reply:
I kept this block for completeness so that users can copy paste these blocks to launch a composite alarm with actions. Without this block users would have to code SNS Topics themselves and realise that AWSCC does not support SNS topics today.